src/include/catalog/database_catalog.h

```diff
@@ -344,6 +348,12 @@ class DatabaseCatalog {
   storage::index::Index *constraints_index_index_;
   storage::index::Index *constraints_foreigntable_index_;
 
+  storage::SqlTable *languages_;
+  storage::index::Index *languages_oid_index_;
+  storage::index::Index *languages_name_index_;  // indexed on language OID and name
```

Comment:
Comment is not accurate.  It looks like the name index is just on the name string.